### PR TITLE
Remove initializations with nil

### PR DESCRIPTION
**Pull request by JeroenDeDauw**
IDE told me those are evilll

### CODE REVIEW

File: lua/ui/game/economy.lua [hunk 1]
```diff
@@ -3,7 +3,7 @@
 --* Author: Chris Blackwell
 --* Summary: Economy bar UI
 --*
---* Copyright � 2005 Gas Powered Games, Inc.  All rights reserved.
+--* Copyright � 2005 Gas Powered Games, Inc.  All rights reserved.
```
Comment (JeroenDeDauw) [inline lua/ui/game/economy.lua]: huh

**Comment by Sheeo** [comment]
They aren't evil -- just explicit, since `local a` will assign `nil` to `a` by default.

**Comment by JeroenDeDauw** [comment]
IDE lists this as perf issue, which is kinda odd

perf

**Comment by Sheeo** [comment]
Well it will likely result in an extra initialization, so it could be a performance issue in a tight loop.

**Pull request closed without merging**